### ISSUE TITLE
OpenAPI spec missing for "TonieCard"

### ISSUE THREAD

**Issue by H3npi** [toniebox-reverse-engineering/teddycloud_web#4]
Within this commit https://github.com/toniebox-reverse-engineering/teddycloud_web/commit/b6315c5e74b7df382e7b93fd30d711ed967a6008 some modifications to the generated client were done.

Trying to add something to the openApi spec and generating a new client will now allways need manual merging.

**Comment by SciLor** [comment]
Yes, I did this, before understanding what this is. It should be moved to a different place, so it doesn't interfere with the OpenApi gen
